libraries/rating-bar/src/main/res/layout/view_rating_bar.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+            name="viewState"
+            type="com.trendyol.uicomponents.ratingbar.RatingBarViewState" />
+    </data>
+
+    <merge
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        tools:parentTag="android.widget.LinearLayout">
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:id="@+id/star_1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_stars"
+            app:srcCompat="@drawable/ic_star"
```

Comment:
Need to add thumbDrawable attr to get drawable instead of forcing default drawable (ic_star.xml)

Reply:
Planning to implement that type of customizations on later releases.